### ISSUE TITLE
at-function-named-arguments: ignoreFunctions setting is ignored

### ISSUE THREAD

**Issue by jens-duttke** [kristerkari/stylelint-scss#451]
I have the following rule setup:
```
'scss/at-function-named-arguments': ['never', { ignoreFunctions: ['scale-color'] }],
```
So I expect, the named argument `$lightness` in the following `scale-color` function call is allowed:
```
div {
  color: scale-color(#fff, $lightness: -75%);
}
```

But it throws:
```
Unexpected a named parameter in function call (scss/at-function-named-arguments) [error]
```

I'm using:
```
"stylelint": "13.0.0",
"stylelint-scss": "3.14.2",
```

**Comment by kristerkari** [comment]
Hmm.. that's weird because this test is passing for me:

<img width="686" alt="Skärmavbild 2020-02-05 kl  09 36 37" src="https://user-images.githubusercontent.com/993108/73820750-169d6a80-47fb-11ea-8c11-cffaa9fd61e6.png">

**Comment by jens-duttke** [comment]
Sorry, I've just realized that it's similar to the problem in #450 (regarding the `operator-no-unspaced` rule), I've reported yesterday.

This issue also only occur if the function call is wrapped in `#{` and `}`.

So, this code throws the error:
```scss
.b {
  --my-var: #{scale-color(#fff, $lightness: -75%)};
}
```

**Comment by airyboy** [comment]
I had a look at it. There is an obvious bug in **_postcss-value-parser_** - it stumbles on interpolation values and returns the name of a function not omitting interpolation braces (_#{scale-color_)

**Comment by airyboy** [comment]
Well, it's not a **_postcss-value-parser_** bug, sorry. the library is not meant to be working with SCSS at all, so it needs to fix this in **_stylelint-scss_**
